src/client/internal.ts

```diff
@@ -19,14 +19,17 @@ import {FDC3Error} from './errors';
  * The identity of the main application window of the service provider
  */
 export const SERVICE_IDENTITY = {
-    uuid: 'fdc3-service',
+    uuid: 'fdc3-service',  // default value
```

Comment:
Wonder if it would be more helpful to slightly change the block comment above this, rather than have this line comment. Suggests only `uuid` can change, when `name` can also change